spec/examples/validations/html.spec.js

```diff
@@ -9,9 +9,12 @@ import assertPassesAcceptance from './assertPassesAcceptance';
 
 import html from '../../../src/validations/html';
 
-function htmlWithBody(body) {
+function htmlWithBody(body, head) {
```

Comment:
I don’t think there’s any need for the `head` argument—we don’t use it do we? I’m fine just always having that be hard-coded.

Also, hate to be the bearer of bad news, but… the test suite has moved! Can you rebase/merge master and throw these tests into `test/unit/validations`? I think you’ll find the actual test code quite nice, sorry for the churn though : (